setup.py

```diff
@@ -81,6 +81,15 @@
                                    "numba/_helperlib.c",
                                    "numba/mathnames.inc"])
 
+ext_lapack = Extension(name="numba._lapack",
+                          include_dirs=[numpy.get_include()],
+                          sources=["numba/_lapack.c"],
```

Comment:
If you're including `_lapack.c` from `_helpermod.c`, why are you also creating a separate extension module from it?